tests/practice/locators-actions.spec.ts

```diff
@@ -0,0 +1,62 @@
+import {test, expect, frameLocator, FrameLocator} from "@playwright/test";
+import {fa, faker} from '@faker-js/faker';
+
+// Test 1 Register form
+test('Register form', async ({ page }) => {
+    await page.goto('https://qa-practice.netlify.app/register');
```

Comment:
why no POM here?

Reply:
@ybaranovskyi11 we should not integrate POM in this homework (Playwright Locators&Actions HW), but in the next task (Playwright Fixtures HW) we should integrate POM

Reply:
@ybaranovskyi11 fixed
![image](https://github.com/user-attachments/assets/6bba49e3-519b-44cf-8b39-8761b046b46b)